### PR TITLE
ksh 2020.0.0

### PR DESCRIPTION
**Pull request by siteshwar**
New stable release of ksh primarily features:

- Switch to new and saner build system
- Hundreds of bug fixes
- Better test coverage

More details can be found at:

https://groups.google.com/d/msg/korn-shell/-tQkll8BxXU/X4ON61CHBwAJ

It is already rebased in Fedora, Debian, Gentoo, FreeBSD and several
other well known distros.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by krader1961** [comment]
FWIW, I've probably contributed more bug fixes and enhancements to ksh in the past couple of years than anyone else. I'd love to see this PR merged so that Homebrew users can easily obtain an updated version of the `ksh` shell. I can't speak to the correctness of this PR but am confident that something like this will provide a much improved `ksh` command to Homebrew users.

File: Formula/ksh.rb [hunk 1]
```diff
@@ -1,47 +1,23 @@
...
-    cellar :any_skip_relocation
-    rebuild 3
-    sha256 "f293f9efa89e7b6cb6318ac6406e61de92b05b85f9c1fd17affccb38e941b64c" => :catalina
-    sha256 "d6a44587ba5c9ef60ccd4df40bb4e5b4be2a6fbf6c3269d1732ba7376b341c8c" => :mojave
-    sha256 "5ca742b83c55b817a0fa09f737d73530c0f73d89618c60a19ec2ea5251ef8cd4" => :high_sierra
-    sha256 "37a9f6c57702896fabe762fe90ebe454f04f3523aea311a28d34dc884d53736e" => :sierra
-    sha256 "5148e18444c7f1a4e7b71f72982362491aa5581101296acaa6d9c2a782d620b1" => :el_capitan
-    sha256 "13f85c7df7f44b68f1e5560c05b61eff8145230d94986537bdee5702c1e72e68" => :yosemite
-    sha256 "7c665466fb323fc0cb6ffb87ec9fe1630d75afcd3b12864e2424677473db4924" => :mavericks
-  end
```
Comment (Moisan) [inline Formula/ksh.rb]: Please do not remove the `bottle` block, our merging process will update it automatically.

**Review by Moisan — CHANGES_REQUESTED** [review]
Outside of the `bottle` block, looks good to me.

**Comment by siteshwar** [comment]
> Outside of the `bottle` block, looks good to me.

Fixed.

**Review by chenrui333 — APPROVED** [review]

**Comment by fxcoudert** [comment]
The caveats are not necessary, and the `init` resource was unused.

**Pull request closed without merging**